packages/table/src/spectrum-table.css

```diff
@@ -0,0 +1,2248 @@
+    .spectrum-Table-row.is-drop-target:before {
+    right: 0; /* [dir=rtl] .spectrum-Table-body.is-drop-target:before,
+   * [dir=rtl] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+:host([dir='ltr']) .spectrum-Table-row.is-drop-target:before,
+:host([dir='ltr'][density='compact']) .spectrum-Table-row.is-drop-target:before,
+:host([dir='ltr'][density='spacious'])
+    .spectrum-Table-row.is-drop-target:before {
+    right: 0; /* [dir=ltr] .spectrum-Table-body.is-drop-target:before,
+   * [dir=ltr] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=ltr] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+:host([dir='rtl']) .spectrum-Table-row.is-drop-target:before,
+:host([dir='rtl'][density='compact']) .spectrum-Table-row.is-drop-target:before,
+:host([dir='rtl'][density='spacious'])
+    .spectrum-Table-row.is-drop-target:before {
+    left: 0; /* [dir=rtl] .spectrum-Table-body.is-drop-target:before,
+   * [dir=rtl] 
+.spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * [dir=rtl] 
+.spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+}
+.spectrum-Table-row.is-drop-target:before,
+:host([density='compact']) .spectrum-Table-row.is-drop-target:before,
+:host([density='spacious']) .spectrum-Table-row.is-drop-target:before {
+    bottom: 0;
+    content: ''; /* .spectrum-Table-body.is-drop-target:before,
+   * .spectrum-Table-row.is-drop-target::before,
+   * .spectrum-Table--compact .spectrum-Table-body.is-drop-target::before,
+   * .spectrum-Table--compact .spectrum-Table-row.is-drop-target::before,
+   * .spectrum-Table--spacious .spectrum-Table-body.is-drop-target::before,
+   * .spectrum-Table--spacious .spectrum-Table-row.is-drop-target::before */
+    position: absolute;
+    top: 0;
+    z-index: 1;
+}
+::slotted(sp-table-body) {
+    border-radius: var(--spectrum-table-regular-border-radius);
+    border-width: var(--spectrum-table-regular-border-size);
+    overflow: auto;
+    position: relative; /* .spectrum-Table-body */
+    vertical-align: var(--spectrum-table-regular-cell-vertical-alignment);
+}
```

Comment:
I'm not sure I like this now that we're not extending Virtualizer to make TableBody...